book/forms.rst

```diff
@@ -344,17 +344,6 @@ object.
 
 .. configuration-block::
 
-    .. code-block:: yaml
```

Comment:
From what I noticed annotations are used much more often for validation (and you probably already use ORM/ODM annotations for your entities) so I think it makes more sense to display them first